src/main/java/org/cryptomator/linux/keychain/SecretServiceKeychainAccess.java

```diff
@@ -15,24 +16,19 @@
 
 	@Override
 	public boolean isSupported() {
-		try (@SuppressWarnings("unused") SimpleCollection keyring = new SimpleCollection()) {
-			// seems like we're able to access the keyring.
-			return true;
-		} catch (IOException | RuntimeException e) {
-			return false;
-		}
+		return SimpleCollection.isAvailable();
```

Comment:
As already stated, this method does not work. See https://github.com/cryptomator/integrations-linux/pull/1#issuecomment-770220325 on how to fix this in your library.

Reply:
This is fixed now. I tested it with Kubuntu `20.04.1`. I also had to adapt another internal call to actually see, if the session can be opened.

Reply:
I took that approach and had to adapt some internal stuff to not run into a `NullPointerException`.